src/app/pages/MediaAssetPage/index.stories.jsx

```diff
@@ -24,6 +24,11 @@ export default {
   Component,
   title: 'Pages/Media Asset Page',
   decorators: [story => <WithTimeMachine>{story()}</WithTimeMachine>],
+  parameters: {
+    chromatic: {
+      delay: 15000,
```

Comment:
Hmm this doesn't seem to have helped as the flake is showing up on this PR still; should I revert this change?